applauncher/004-three-by-three.qml

```diff
@@ -45,68 +46,15 @@ Item {
     property bool dragStop: false
     property string appTitle: ""
     property real clickY: 0
+    property int numColumns: 3
+    
+    Component {
+        id: gridDelegate
 
-    GridView {
-        id: appsView
-
-        flow: GridView.FlowLeftToRight
-        snapMode: GridView.SnapToRow
-        anchors {
-            fill: parent
-            leftMargin: parent.width * .026
-            rightMargin: parent.width * .026
-        }
-        clip: true
-        cellHeight: appsView.height / 3.15
-        cellWidth: appsView.width / 3
-        preferredHighlightBegin: height / 3.15 - currentItem.height / 3.15
-        preferredHighlightEnd: height / 3.15 + currentItem.height / 3.15
-        highlightRangeMode: ListView.StrictlyEnforceRange
-        contentY: -(height / 3.15 - (height / 6.3))
-
-        property int currentPos: 0
-
-        onCurrentPosChanged: {
-            rightIndicator.animate()
-            leftIndicator.animate()
-            topIndicator.animate()
-            bottomIndicator.animate()
-        }
-
-        onDragStarted: {
-            dragStop = true
-        }
-
-        Connections {
-            target: grid
-            function onCurrentVerticalPosChanged() {
```

Comment:
This is also linked to a global variable and makes sure that if you: open an app, let the watch go to sleep, unlock, go back to the launcher, you are at the top of the scroll view rather than halfway through it. It should also be kept

Reply:
Can confirm that the 'go to home gesture' leaves the app launcher at a the previous location, which isn't intended for these vertical app launchers.

Reply:
I have restored it, but the behavior I observe is the same with or without it, so I'll leave it to someone else to resolve.  Philosophically, I don't know why the horizontally organized launchers keep their place (which I find convenient) but the vertically organized ones should not.